src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RankSumTest.java

```diff
@@ -79,7 +79,7 @@
     }
 
     protected void fillQualsFromLikelihood(VariantContext vc, ReadLikelihoods<Allele> likelihoods, List<Double> refQuals, List<Double> altQuals, int refLoc) {
-        for (final ReadLikelihoods<Allele>.BestAllele bestAllele : likelihoods.bestAlleles()) {
+        for (final ReadLikelihoods<Allele>.BestAllele bestAllele : likelihoods.bestAllelesBreakingTies()) {
```

Comment:
If fillQualsFromLikelihood doesn't change, then why do the rank sum values change in some of the variants in your new expected outputs?  It still excludes uninformative reads, but your new searchBestAllele has _made_ them informative.

Reply:
I need to figure that out.  The new code in `searchBestAllele` is
```
if ( bestLikelihood - referenceLikelihood < BestAllele.INFORMATIVE_THRESHOLD ) {
    secondBestLikelihood = bestLikelihood;
    bestAlleleIndex = referenceAlleleIndex;
    bestLikelihood = referenceLikelihood;
}
```
and so the `BestAllele` constructor gets a best likelihood of `referenceLikelihood` and a second-best likelihood of `bestLikelihood`.  The confidence is then `referenceLikelihood - bestLikelihood`, which is negative and thus less than the informative threshold.

It's also puzzling why the other rank sums don't change.  I'll have to think some more.

Reply:
Ah, I think I've got it.  The new Smith-Waterman parameters affect the pileup for some informative reads when they turn deletions into base errors, which shifts some bases into/out of some pileups.

Reply:
Yep, I just came to the same conclusion.  I think changing the pileups is fine.

One last thing: do the likelihoods for uninformative reads get changed more than once?  There are multiple calls to searchBestAllele on the same likelihoods -- e.g. each rank sum test.

Reply:
The likelihoods themselves are never modified, just the `BestAllele`s that are determined from the likelihoods.  That is, every annotation sees the same `ReadLikelihoods` as before the change, and every annotation calculates the `BestAllele` for each read afresh.

Reply:
For some reason I thought bestLikelihood and the like were member variables, but clearly they are not, in which case this is fine.